integration-testing/src/functionalTest/kotlin/kotlinx.atomicfu.gradle.plugin.test/framework/runner/BuildRunner.kt

```diff
@@ -19,9 +19,11 @@ internal class GradleBuild(val projectName: String, val targetDir: File) {
             add("-P$ENABLE_JVM_IR_TRANSFORMATION=$enableJvmIrTransformation")
             add("-P$ENABLE_JS_IR_TRANSFORMATION=$enableJsIrTransformation")
             add("-P$ENABLE_NATIVE_IR_TRANSFORMATION=$enableNativeIrTransformation")
-            localRepositoryUrl?.let {
-                add("-P$LOCAL_REPOSITORY_URL_PROPERTY=$it")
-            }
+            // Pass kotlin_version and kotlin.native.version parameters used for the library build to the builds of integration tests.
+            // These versions may be overriden by the TC build config.
+            if (kotlinVersion.isNotEmpty()) add("-P$KOTLIN_VERSION_PARAMETER=$kotlinVersion")
+            if (kotlinNativeVersion.isNotEmpty()) add("-P$KOTLIN_NATIVE_VERSION_PARAMETER=$kotlinNativeVersion")
+            localRepositoryUrl?.let { add("-P$LOCAL_REPOSITORY_URL_PROPERTY=$it") }
```

Comment:
can we get rid of `localRepositoryUrl` using `kotlinArtifactsRepo` instead of this?

Reply:
There is one test (`user-project`), that is supposed to be built against the library that uses atomicfu. So `user-project` depends on `mpp-project`, that is published to `maven("build/.m2/")`. And in this case I pass `"build/.m2/" `as a `localRepositoryUrl`